internal/armada/server_test.go

```diff
@@ -177,7 +181,8 @@ func withRunningServer(action func(client api.SubmitClient, leaseClient api.Aggr
 	if err != nil {
 		panic(err)
 	}
-	defer minidb.Close()
+	// minidb.Close hangs indefinitely (likely due to a bug in miniredis)
+	// defer minidb.Close()
```

Comment:
Is this a good idea? Did you try newer versions of miniredis or do they have breaking api changes that makes upgrading non-trivial?

Reply:
I don't think it matters much for us. Since the server is cleaned up when the process exits.